plugins/cuda/cuda_plugin.c

```diff
@@ -116,7 +116,8 @@ static int launch_cuda_checkpoint(const char **args, char *buf, int buf_size)
 			return -1;
 		}
 		close(fd[READ]);
-		return execvp(args[0], (char **)args);
+		execvp(args[0], (char **)args);
```

Comment:
I think we need to print  error/warning here.

Reply:
In the child process STDOUT and STDERR are set to a pipe fd. However, we handle the exit code of the process and show appropriate messages. I've updated the log messages so that when cuda-checkpoint is not installed, CRIU will show the following debug message:
```
cuda_plugin: cuda-checkpoint not present in $PATH
```

Reply:
logfd isn't STDOUT or STRERR, is it?

Reply:
yes, but for some reason it appears to be closed after `execvp()`.
Should we use `close_fds()` here to avoid leaking file descriptors (e.g., https://github.com/checkpoint-restore/criu/commit/6c8ea60491305482eb60d9b17a8635ab9248a5ab)?

Reply:
I've updated the pull request to call `close_fds()` before `execvp()` and to show the following messages when `cuda-checkpoint` is not installed:
```
(00.008034) cuda_plugin: execvp("cuda-checkpoint") failed: No such file or directory
(00.008055) cuda_plugin: cuda-checkpoint not present in $PATH
```